microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -751,5 +756,27 @@ See also:
 * [Pause code with breakpoints](../javascript/breakpoints.md)
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 109.
-*  This checkbox is present in Microsoft Edge Stable 107.
+*  This checkbox is present in Microsoft Edge Canary 111.
+*  This checkbox is present in Microsoft Edge Stable 108.
+
+
+<!-- ====================================================================== -->
+## Enable Preloading Status Panel in Application panel
+
+_new as of Microsoft Edge 110_
+
+Adds the **Preloading** section within the **Application** tool.  The **Preloading** section of the list of pages contains a **Prefetching & Prerendering** page, which contains columns:
+* **Started at**
+* **Type**
+* **Trigger**
+* **URL**
+* **Status**
+
+Select an element for more details.
+<!-- todo: how select element, where? -->
```

Comment:
@captainbrosset - any clarification?  Enable Preloading Status Panel in Application panel

GitHub rendered section has picture:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/whats-new-109/microsoft-edge/devtools-guide-chromium/experimental-features/index.md#enable-preloading-status-panel-in-application-panel

Reply:
This is related to this new feature: https://developer.chrome.com/blog/prerender-pages/
In summary, in certain cases (either when the browser decides, or if the page uses a special API), the next page that the user will visit is pre-rendered in the background, in order to speed up page navigation.

This new panel will eventually (I think it's still too early to be able to fully use it) display the list of pages that got pre-rendered by the browser and why. This way, developers can debug their code that uses this new pre-render special API.

Reply:
added text